src/store/item-store.js

```diff
@@ -32,4 +36,12 @@ store.registerHandler('ITEMS_RECEIVED', data => {
     items = data.items;
     store.emitChange();
 });
+
+store.registerHandler('STUDENT_FOUND', data => {
+    items = data.items;
+    for(let item of data.items) {
+        item.isOverdue = store.isOverdue(item);
```

Comment:
Similar to the `student.hasOverdueItem` issue here... We could store `false` here now, but the value would be stale if the item becomes overdue during this time.